### PR TITLE
Add column definition

### PR DESCRIPTION
**Pull request by srikanthccv**
This PR adds a `Column` that is used to describe the CH column in code. This will be used to implement all the operations on column.

### CODE REVIEW

**Comment by srikanthccv** [comment]
This should be easy one.

File: cmd/signozschemamigrator/schema_migrator/column.go [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+	}
+	if c.Settings != nil {
+		sql.WriteString(" SETTINGS ")
+		sql.WriteString(c.Settings.String())
+	}
+	if c.Comment != "" {
+		sql.WriteString(" COMMENT ")
+		sql.WriteString(c.Comment)
+	}
+	return sql.String()
```
Comment (nityanandagohain) [inline cmd/signozschemamigrator/schema_migrator/column.go:277]: add few tests for these functions ?  or you want to address tests separately.

Comment (srikanthccv) [reply cmd/signozschemamigrator/schema_migrator/column.go:277]: The tests are coming in the upcoming PR where the column definition is used.

**Review by nityanandagohain — APPROVED** [review]
Looks good to start with,  we can add more if anything if required.

**Pull request merged**